components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -2211,7 +2211,7 @@ public String getClientType() {
                                 + "'");
             }
             if (fromDate != null && toDate != null) {
-                String granularity = APIUsageStatisticsClientConstants.MINUTES_GRANULARITY;//default granularity
+                String granularity = APIUsageStatisticsClientConstants.SECONDS_GRANULARITY;//default granularity
```

Comment:
does this mean if any range is not selected, default graphs will show seconds related data?